webDriverIO/specs/data/list-users-schema.ts

```diff
@@ -0,0 +1,54 @@
+const userSchema = {
```

Comment:
I'd suggest defining these schemas using the built-in options of the JSON Schema (e.g., definitions, references, etc.).